scripts/pull_backend_config_from_s3

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
```

Comment:
This file looks mostly like the one in `.circleci`, right? Can we remove the copy in `.circleci`, and just use this one?